libfoundation/src/system-file-posix.cpp

```diff
@@ -24,6 +24,11 @@ along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
 #include <sys/stat.h>
 #include <errno.h>
 #include <dirent.h>
+#include <dlfcn.h>
+
+#ifdef _MACOSX
+#include <mach-o/dyld.h>
+#endif
 
```

Comment:
Just left over from when I first implemented the module loading stuff in that file :-) will remove